test/nuts/mdapi.nut.ts

```diff
@@ -58,6 +60,38 @@ describe('mdapi NUTs', () => {
     });
   });
 
+  describe('mdapi:deploy:cancel', () => {
+    it('will cancel an mdapi deploy via the stash.json', () => {
+      execCmd('force:source:convert --outputdir mdapi');
+      // TODO: once mdapi:deploy is migrated switch to execCmd
+      const deploy = JSON.parse(exec('sfdx force:mdapi:deploy -d mdapi -w 0 --json', { silent: true })) as {
+        result: { id: string };
+      };
+      const result = execCmd('force:mdapi:deploy:cancel --json');
```

Comment:
Is there any chance this won't cancel in time (I think I remember that with other NUTs where sometimes it would succeed before the cancel request arrived, depending on how busy the mdapi was).